src/resources/js/events-admin.js

```diff
@@ -146,7 +146,7 @@ Date.prototype.format = function( mask, utc ) {
  */
 jQuery( document ).ready( function( $ ) {
 
-	var $date_format      = $( '[data-datepicker_format]' ),
+	var $date_format      = $( '[data-datepickerFormat]' ),
```

Comment:
Looks like this file still has reference to `datepickerFormat` in 3 places rather than `datepicker_format`.